tests/code_update.py

```diff
@@ -52,6 +53,91 @@ def test_verify_quotes(network, args):
     return network
 
 
+@reqs.description("Test that the security policies table is correctly populated")
+@reqs.snp_only()
+def test_add_node_with_correct_security_policy(network, args):
+
+    LOG.info(f"Checking governance table security policies")
```

Comment:
FYI, the `@reqs.description` message is printed when the test starts so no need for an extra log line here.